src/core/store/actions/globals.js

```diff
@@ -47,3 +47,9 @@ export function acceptTerms() {
     type: constants.ACCEPT_TERMS
   }
 }
+
+export function didUseVideo() {
+  return {
+    type: constants.DID_USE_VIDEO
```

Comment:
hm, this might not be necessary. if when we set the capture on the redux layer, we also pass the variant, then we can always find out and we don't have to  make it a global variable.